server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/resources/cucumber/GetMessages.feature

```diff
@@ -199,3 +199,14 @@ Feature: GetMessages method
     And the preview of the message is "multipart/related content"
     And the property "textBody" of the message is null
     And the htmlBody of the message is "<html>multipart/related content</html>\n"
+
+  Scenario: Retrieving message should return retrieve textBody and htmlBody when incoming mail have an inlined HTML and text body without Content-ID
+    Given the user has a message in "inbox" mailbox with subject "my test subject" and from file "eml/inlinedMultipart.eml"
+    When the user ask for messages "["username@domain.tld|inbox|1"]"
+    Then no error is returned
+    And the list should contain 1 message
+    And the hasAttachment of the message is "true"
```

Comment:
You may remove those checks, only the preview, textBody & htmlBody are interesting in this scenario